diff --git a/server/Dockerfile b/server/Dockerfile
index 58777d5..de2ab96 100644
--- a/server/Dockerfile
+++ b/server/Dockerfile
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8-minimal
+FROM debian:buster-slim
 
 ENV KEYCLOAK_VERSION 13.0.0
 ENV JDBC_POSTGRES_VERSION 42.2.5
@@ -17,7 +17,9 @@ ARG KEYCLOAK_DIST=https://github.com/keycloak/keycloak/releases/download/$KEYCLO
 
 USER root
 
-RUN microdnf update -y && microdnf install -y glibc-langpack-en gzip hostname java-11-openjdk-headless openssl tar which && microdnf clean all
+RUN mkdir -p /usr/share/man/man1
+RUN apt-get update -y && apt install -y curl gzip hostname openjdk-11-jre-headless openssl tar && apt autoremove
+RUN /usr/bin/c_rehash
 
 ADD tools /opt/jboss/tools
 RUN /opt/jboss/tools/build-keycloak.sh
