diff --git a/server/Dockerfile b/server/Dockerfile
index d28336f..aba8e49 100644
--- a/server/Dockerfile
+++ b/server/Dockerfile
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8-minimal
+FROM registry.fedoraproject.org/fedora-minimal:34
 
 ENV KEYCLOAK_VERSION 12.0.1
 ENV JDBC_POSTGRES_VERSION 42.2.5
diff --git a/server/tools/build-keycloak.sh b/server/tools/build-keycloak.sh
index 3bb9379..5d3db2e 100755
--- a/server/tools/build-keycloak.sh
+++ b/server/tools/build-keycloak.sh
@@ -43,8 +43,9 @@ else
     echo "Keycloak from [download]: $KEYCLOAK_DIST"
 
     cd /opt/jboss/
-    curl -L $KEYCLOAK_DIST | tar zx
-    mv /opt/jboss/keycloak-* /opt/jboss/keycloak
+    curl -L $KEYCLOAK_DIST --output keycloak-dist.tar.gz
+    mkdir keycloak
+    tar xzf keycloak-dist.tar.gz -C keycloak --strip-components=1
 fi
 
 #####################
